search/search-client-query-transformer/src/main/java/io/camunda/search/clients/transformers/sort/DecisionRequirementsFieldSortingTransformer.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.search.clients.transformers.sort;
+
+import static io.camunda.webapps.schema.descriptors.operate.index.DecisionRequirementsIndex.KEY;
+
+public class DecisionRequirementsFieldSortingTransformer implements FieldSortingTransformer {
+
+  @Override
+  public String apply(final String domainField) {
+    return switch (domainField) {
+      case "decisionRequirementsKey" -> KEY;
+      default -> domainField;
```

Comment:
🔧  list all possible fields and throw exception (as in `DecisionInstanceFieldSortingTransformer`)

Reply:
yes I just did that before the review :) 
so, I have force pushed the changes